client/css/style.css

```diff
@@ -1498,22 +1497,33 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	content: "Search Results";
 }
 
-#loading {
+#loading.active {
 	font-size: 14px;
 	z-index: 1;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	flex-direction: column;
+	text-align: center;
 }
 
 #loading p {
 	margin-top: 10px;
 }
 
+#loading img {
+	max-width: 256px;
```

Comment:
Would you mind adding a `max-height` or something (Chrome tells me the image shows as 256px x 171.48px)? On load there is a bump until the image is loaded, which is right in the middle of loading (at least in my case).
I'm playing with slow network and it does that too.